arbor/backends/multicore/shared_state.cpp

```diff
@@ -91,10 +103,13 @@ void ion_state::zero_current() {
 
 void ion_state::reset() {
     zero_current();
-    std::copy(reset_Xi_.begin(), reset_Xi_.end(), Xd_.begin());
```

Comment:
Is reading out the ion concentration (in the case of `USEION X READ Xd`) unaffected by this?

Reply:
Yes, this is just for resetting it to the starting value.